Dshop/apps/payments/views.py

```diff
@@ -13,3 +15,68 @@ def stripe_config(request):
     if request.method == 'GET':
         stripe_config = {'publicKey': settings.STRIPE_PUBLISHABLE_KEY}
         return JsonResponse(stripe_config, safe=False)
+
+
+@csrf_exempt
+def create_checkout_session(request):
+    if request.method == 'GET':
+        stripe.api_key = settings.STRIPE_SECRET_KEY
+        product_price = 99.99
+        try:
+            current_site = get_current_site(request)
+            domain = current_site.domain
+            protocol = 'https' if request.is_secure() else 'http'
+            checkout_session = stripe.checkout.Session.create(
+                client_reference_id=request.user.id if request.user.is_authenticated else None,
+                success_url=f'{protocol}://{domain}/payments/success/',
+                cancel_url=f'{protocol}://{domain}/payments/cancelled/',
+                payment_method_types=['card'],
+                mode='payment',
+                # TODO: Product implementation
+                line_items=[
+                    {
+                        'quantity': 1,
+                        'price_data': {
+                            'currency': 'pln',
+                            'unit_amount': int(product_price * 100),
+                            'product_data': {
+                                'name': 'T-shirt',
+                            },
+                        },
+                    }
+                ]
+            )
+            return JsonResponse({'sessionId': checkout_session['id']})
+        except Exception as e:
+            return JsonResponse({'error': str(e)})
+    return JsonResponse({'error': 'Invalid request method'}, status=400)
+
+
+@csrf_exempt
+def stripe_webhook(request):
```

Comment:
Since we try to use Class Based Views could you move it into default VIew? Like here:
https://docs.djangoproject.com/en/4.2/topics/class-based-views/intro/#using-class-based-views

This would be nicely consistent with the rest of the site.